ipgeocache/__main__.py

```diff
@@ -1,21 +1,29 @@
-import os
-import sys
+import click
 
+from .printers import default_printer, json_printer
 from . import get as get_ip_info
 
 
-def main():
-    assert (
-        "IPINFO_TOKEN" in os.environ
-    ), "Couldn't find the IPINFO_TOKEN environment variable!"
-    if len(sys.argv) <= 1:
-        print(
-            "Must provide IP address to geolocate as the first argument!",
-            file=sys.stderr,
-        )
-        sys.exit(1)
-    for k, v in get_ip_info(sys.argv[1]).items():
-        print(f"{k}={v}")
+@click.command()
+@click.option("--ipinfo-token",
+              help="Authentication token to use ipinfo API. Consult https://ipinfo.io/signup", envvar="IPINFO_TOKEN")
+@click.option("--json", is_flag=True)
+@click.argument("ip")
+def main(ip, ipinfo_token, json):
+    """Gets geolocation for an IP
+
+    IP is public internet protocol address to be geolocated
+    """
+    if ipinfo_token is None:
+        click.secho("Couldn't find the IPINFO_TOKEN environment variable!\n"
+                    "Set it on IPINFO_TOKEN or pass from CLI --ipinfo-token", fg="red")
```

Comment:
Sounds a bit strange, to repeat `IPINFO_TOKEN` twice.

I think this could be replaced with:

```
"Could not find ipinfo access token!\n"
"Set the IPINFO_TOKEN environment variable or pass the --ipinfo-token flag
```

Reply:
Thanks, applied your suggestion.
Btw any chance you tag your repo with hacktoberfest tag ?

Reply:
I think I've done that now.